cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -100,6 +102,16 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     super.setStartupUrl("https://www.youtube.com/tv");
+    if (!CommandLine.isInitialized()) {
+      ((ContentShellApplication) getApplication()).initCommandLine();
+      String[] commandLineParams = new String[]{"",
+      "--disable-fre", "--no-first-run",
```

Comment:
This seems fine, but ideally i'd leave either a 
- comment that explains the non-self-explanatory ones ( fre ? ) 
- or a link to a page / doc that explains those

Reply:
Updated  the description, and removed kiosk as chromium on android doesn't support it.

Reply:
I would leave an inline comment so that things would look like 
```
    // Disable "First Run Experience".
   "--disable-fre",
   "--no-first-run",
```

etc

Reply:
updated the comments and rebased this PR.